packages/admin/src/error/errordialog/ErrorDialogProvider.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+
+import { ErrorDialog, ErrorDialogOptions } from "./ErrorDialog";
+import { ErrorDialogContext, ErrorDialogContextProps } from "./ErrorDialogContext";
+
+export const ErrorDialogProvider: React.FunctionComponent = ({ children }) => {
+    const [errorOptions, setErrorOptions] = React.useState<ErrorDialogOptions | undefined>(undefined);
+    const [errorDialogVisible, setErrorDialogVisible] = React.useState(false);
+
+    const errorDialog = React.useMemo((): ErrorDialogContextProps => {
+        return {
+            showError: (options: ErrorDialogOptions) => {
+                setErrorOptions(options);
+                setErrorDialogVisible(true);
+            },
+        };
+    }, [setErrorOptions]);
```

Comment:
`setErrorOptions` does not need to be in the dependencies https://reactjs.org/docs/hooks-reference.html#usestate

Reply:
that's true - i just tested this and due to the documention it's working 👍 

Made a PR: https://github.com/vivid-planet/comet-admin/pull/390